### DISCUSSION TITLE
mitmproxy doesnt show all requests

### DISCUSSION THREAD

**Discussion by DrNokkel** [mitmproxy/mitmproxy#6238] [Q&A]
I use the "PwndPassDLL" to check passwords against the "Have I Been Pwned" database. But I cannot see any traffic between the client and this database in "mitmproxy." 

However, I can see the traffic in Wireshark, as shown in the screenshot below. The traffic is going through the IP address 104.18.206.87.

![grafik](https://github.com/mitmproxy/mitmproxy/assets/139129250/b69e34de-49cd-47b0-9f2e-91d8e83880c6)

**Comment by mhils** [comment]
This sounds like your target application is simply ignoring the proxy settings.
